.config/forest.dic

```diff
@@ -44,6 +44,8 @@ MiB
 multisig
 mutex
 P2P
+parsable
+parseable
```

Comment:
They mean the same thing, right? Let's pick one or the other.

Reply:
Yepp they mean the same thing and both are valid spellings with similar usage frequency. I'll go with `parsable` as [it's the preferred standard form](https://en.wiktionary.org/wiki/parsable).